src/main/webapp/resources/js/components/samples/components/SampleMetadata.jsx

```diff
@@ -9,26 +21,79 @@ import { Empty, List } from "antd";
  * @constructor
  */
 export function SampleMetadata({ metadata }) {
-  return Object.keys(metadata).length ? (
-    <List
-      itemLayout="horizontal"
-      dataSource={Object.keys(metadata).sort((a, b) => a.localeCompare(b))}
-      renderItem={(item) => (
-        <List.Item className="t-sample-details-metadata-item">
-          <List.Item.Meta
-            title={
-              <span className="t-sample-details-metadata__field">{item}</span>
-            }
-            description={
-              <span className="t-sample-details-metadata__entry">
-                {metadata[item].value}
-              </span>
-            }
+  const [visible, setVisible] = React.useState(false);
+  return (
+    <>
+      <Button onClick={() => setVisible(true)}>
+        {i18n("SampleMetadata.addNewMetadata")}
+      </Button>
+      <Modal
+        className="t-add-metadata-field"
+        onCancel={() => setVisible(false)}
+        visible={visible}
+        onOk={() => setVisible(false)}
+      >
+        <Title level={4}>{i18n("SampleMetadata.modal.title")}</Title>
+        <Form>
```

Comment:
Can you make the layout on this vertical so that the labels are on top of the inputs (like every single other form on this project ;)).

Reply:
Sounds good